backend/gmailAuth.js

```diff
@@ -81,24 +73,23 @@ function getNewToken(oAuth2Client, callback) {
 }
 
 function makeBody(to, from, subject, message) {
-  var str = ["Content-Type: text/plain; charset=\"UTF-8\"\n",
-    "MIME-Version: 1.0\n",
-    "Content-Transfer-Encoding: 7bit\n",
-    "to: ", to,"\n",
-    "from: ", from, "\n",
-    "subject: ", subject, "\n\n",
+  const email =
```

Comment:
- `to` variable is never used in the string
- `from` field is set twice in this string